extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -352,6 +352,12 @@ public static function onLinkEnd( $skin, Title $target, array $options, &$text,
 			$attribs['rel'] = 'nofollow';
 		}
 
+		if ( strpos( $attribs['href'], 'redlink=1' ) !== false ) {
```

Comment:
We add `redlink=1` when there is `'broken'` value in `$options`, see https://github.com/Wikia/app/blob/98d2c8db4d82daaa6b7d3634a0029d8370c5332f/includes/Linker.php#L300-L307
There is no need for additional check, you can just move the actions inside this block to the block above.